eureka_ml_insights/metrics/aime_metrics.py

```diff
@@ -0,0 +1,15 @@
+from tqdm.auto import tqdm
+
+from eureka_ml_insights.metrics.metrics_base import ClassicMetric
+
+
+class AIME_ExactMatch(ClassicMetric):
```

Comment:
Can we use the existing ExactMatch metric that differentiates between invalid and wrong?

Reply:
Also, ClassicMetric always assumes the answer is in model_output column (which here contains the whole response. Since the answer is being extracted into another column here we should either i) update classic metric to accept custom columns or  ii) extract the answer into model_output.